sormas-backend/src/main/java/de/symeda/sormas/backend/common/AbstractDomainObject.java

```diff
@@ -181,6 +181,15 @@ public int hashCode() {
 		return getUuid().hashCode();
 	}
 
+	public String caption() {
+		return toString();
+	}
+
+	@Override
+	public String toString() {
```

Comment:
Superfluous default implementation, but `getClass() + StringUtils.SPACE + getUuid()` would remove a lot of implementation on subclasses. Since this should only appear rarely and in log files, I would not care about the `"Entity"Dto.I18N_PREFIX`.